packages/dds/tree/src/feature-libraries/optional-field/optionalFieldChangeTypes.ts

```diff
@@ -43,14 +44,14 @@ export interface OptionalChangeset<TChildChange = NodeChangeset> {
 	 *
 	 * Rebasing logic should only generate moves whose `src` is an occupied register.
 	 */
-	moves: readonly Move[];
+	readonly moves: readonly Move[];
```

Comment:
comment needs an update. in general probably members of this interface do.

Reply:
it'd probably be helpful to move some subset of this near codecs or persisted types, so that the alternative format (which we didn't want to break) has some docs on semantics.